bundle/metadata/annotations.yaml

```diff
@@ -15,4 +15,4 @@ annotations:
   operators.operatorframework.io.test.config.v1: tests/scorecard/
 
   # Annotations to specify supported OCP versions.
-  com.redhat.openshift.versions: v4.15-v4.16
+  com.redhat.openshift.versions: "v4.15"
```

Comment:
Main branch has:
```
  com.redhat.openshift.versions: "v4.16-v4.17"
 ```
operator 1.6.0 has 
```  com.redhat.openshift.versions: v4.15-v4.16 ```
Are we dropping 4.16 for the patch only? If this is due to yaml fix, we probably want to adopt what main has